examples/pyoa_simpletest.py

```diff
@@ -1,11 +1,13 @@
+#pylint disable=wrong-import-position
+
 import sys
 sys.path.append('/Adafruit_CircuitPython_PYOA')
```

Comment:
you can remove this, its for testing

Reply:
Fixed, and Travis is green. Have another look.